site/README.md

```diff
@@ -0,0 +1,12 @@
+# About
+Source code for the https://devswag.io website.
+
+# Running locally
+1. Clone this repository by running 
+```
+git clone https://github.com/swapagarwal/swag-for-dev.git
+```
+2. Ensure you are running an up-to-date version of [Node.js](https://nodejs.org/en/) on your machine, and that you have npm installed.
+3. Make sure you have [Gulp](https://gulpjs.com/) installed by running `npm install gulp -g`
```

Comment:
Please no ! They have to install globaly `gulp-cli`, not `gulp` which is already provided by the local package.json.

Reply:
Whoops, my bad. Should I change it to `npm install gulp-cli -g`, or just omit it since it's already in package.json?

Reply:
The CLI is required to use Gulp, so yeah, they need to install `gulp-cli` globally as you wrote.

Reply:
Alright, I'll change it to that. Thanks!